src/woo_publications/contrib/documents_api/api.py

```diff
@@ -0,0 +1,78 @@
+"""
+Implement a placeholder/mock "Catalogi API" endpoint for informatieobjecttype resource.
+
+The real implementation will translation the
+:class:`woo_publications.metadata.models.InformatieCategorie` into something that is
+API standard compliant.
+"""
+
+from django.urls import path
+from django.utils.translation import gettext_lazy as _
+
+from drf_spectacular.utils import extend_schema
+from rest_framework import permissions, serializers, views
+from rest_framework.request import Request
+from rest_framework.response import Response
+from zgw_consumers.api_models.constants import VertrouwelijkheidsAanduidingen
+
+DUMMY_IOT = {
+    "omschrijving": "Placeholder",
+    "vertrouwelijkheidaanduiding": VertrouwelijkheidsAanduidingen.openbaar,
+    "begin_geldigheid": "2024-09-01",
+    "concept": False,
+    "informatieobjectcategorie": "Placeholder",
+    "besluittypen": [],
+    "zaaktypen": [],
+}
+
+
+class CatalogiAPIDocumentTypeSerializer(serializers.Serializer):
```

Comment:
Can we add some help_texts? I know that this is temporary but it is still nice for the documentation.
And mark the required fields as such.

Reply:
I'm only implementing the required fields, otherwise Open Zaak complains :) the default in DRF is `required=True`